src/Components/App/App.js

```diff
@@ -2,24 +2,37 @@ import movieData from '../../sampleData'
 import Movies from '../Movies/Movies'
 import './App.css'
 import { Component } from 'react'
+import selectedMovieData from '../../sampleData2'
+import MovieDetails from '../MovieDetails/MovieDetails'
 const logo = require('../../Images/cherry-tomato.png')
 
 class App extends Component {
   constructor() {
     super() 
     this.state = {
-      movies: movieData
+      movies: movieData,
+      singleMovie: ""
```

Comment:
Great call updating this.state.singleMovie to be an empty string on page load so that our source of truth is accurate on page load.